lib/browser/rpc-server.ts

```diff
@@ -94,9 +92,9 @@ const getPreloadScript = async function (preloadPath) {
   return { preloadPath, preloadSrc, preloadError };
 };
 
-ipcMainUtils.handleSync('ELECTRON_BROWSER_SANDBOX_LOAD', async function (event) {
-  const preloadPaths = event.sender._getPreloadPaths();
-  const webPreferences = event.sender.getLastWebPreferences() || {};
+ipcMainUtils.handleSync('ELECTRON_BROWSER_SANDBOX_LOAD', async function (event: IpcMainInvokeEvent) {
+  const preloadPaths = (event.sender as any)._getPreloadPaths() as string[];
+  const webPreferences = (event.sender as any).getLastWebPreferences() || {};
```

Comment:
shouldn't these be declared in typings/internal-electron.d.ts?

Reply:
yep, but they aren't :P